src/Host/Client/Impl/Host/IBrokerClient.cs

```diff
@@ -11,6 +11,7 @@ public interface IBrokerClient : IDisposable {
         string Name { get; }
```

Comment:
Well, looks like an idea to merge these two interfaces was a bit reckless.
- IConnectionInfo has a `Path` property, which is a string. It is good for both UI and settings storage. We actually don't want to replace it with URL because we want to preserve user input after he has saved it.
- IBrokerClient has a `Uri` property, which is URI. It is good for web service operation. Also, on the level of `Host.Client` assembly we don't care about user input format. 

Equality logic is also different:
- IConnectionInfo instances are considered equal if they have identical names.
- IBrokerClient instances are considered equal if they have equal Id's.

So I'd rather not touch it for now, cause it is not that simple and also we introduce new bugs.